tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = flake8
+
+[testenv:flake8]
+deps=flake8
+commands=flake8 --ignore=F403,E221,E501,F405 library
```

Comment:
Well if the command looks like you are supposing there is a problem with that ansible modules code-style policy is more benevolent comparing to standard python (line length for example) so there will be many errors shown.

Another problem is that ansible modules require a star import which will cause many more flake8 complains (I hate ansible for this).

If done so, the resulting exit code of tox will be nonzero -- tests didn't pass.

If we use `library/manageiq.py`, we will test only the `manageiq.py` file even though there may be more of them in the future.